src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,13 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+interface Props {
+  todos: Todo[];
+  isModal: number;
```

Comment:
Consider using a more descriptive name for the `isModal` prop, such as `activeTodoId`, to better convey its purpose and improve code readability.